packages/bpk-component-graphic-promo/src/BpkGraphicPromoTextAlign.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+const TEXT_ALIGN = {
```

Comment:
Given the size of this export we can probably just include this inside the `BpkGraphicPromo.js` file and export from there!

Reply:
Yes, happy to do that. I just tried to follow what the other components are doing in the codebase so as not to break the conventions. See `packages/bpk-component-image/src/BpkImageBorderRadiusStyles.js`. If this is not a convention, then I am more than happy to put it in a single file.

Reply:
Yeah that component isn't probably the best example to take from as its in need of an update at some point. So would have more preference to put in one file given the number of lines this export is